conftest.py

```diff
@@ -165,14 +169,25 @@ def run_js(self, code):
                             throw new Error(`Python exited with error flag set!`);
                         }
                     }
+           """
+        else:
+            check_code = ""
```

Comment:
It's true that the above code snippet is already not run if pyodide is not loaded. However I still find it useful to be able to run some JS code with all interaction with pyodide disabled.

Reply:
Yeah I agree that it is useful. For instance I was running into some issues with this stuff when testing the fatal errors because after the fatal error nothing works. We might consider adding a `mark` to disable all checks.